etc/org.opencastproject.liveschedule.impl.LiveScheduleServiceImpl.cfg

```diff
@@ -10,7 +10,7 @@
 #live.streamingUrl=http://streaming.server/hls/
 
 # If a comma-separated list is provided, several resolutions will be generated for each flavor
-live.resolution=1920x540,960x270
+live.resolution=1920x1080,1280x720
```

Comment:
Hi @LukasKalbertodt, it might help adopters to have a comment regarding the background of these numbers... something like
```
# The default resolution is 16:9, a typical single movie aspect ratio
# If your default is to stream two 16:9 images together, side by side, it may help to expand the resolution to 32:9. For example 1920x540,960x270
```

Reply:
Oh, now I finally understand where this weird resolution comes from :) Makes sense! I amended my commit to contain (almost) the comment you suggest.